wagtailio/services/templates/services/blocks/section.html

```diff
@@ -0,0 +1,46 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+{% load static %}
+{% with value as section %}
+<section id="{{ section.title|slugify }}" class="section section--{{ section_modifier }}">
+    <div class="section__inner-wrapper">
+        {% if section.section_media or section.section_image %}
+            <div class="section-image__wrapper">
+                <div class="section-image">
+                    <div class="section-image__overlay section-image__overlay--var1"></div>
+                    <div class="section-image__overlay section-image__overlay--var2"></div>
+                    {% if section.section_media %}
+                        <div class="section-image__video-container">
+                            <div class="section-image__video-wrapper">
+                                <video autoplay="autoplay" loop="loop" muted="muted">
```

Comment:
`autoplay="autoplay"` - i've not seen this before but seen that it is a valid attribute, what an odd choice of name!

Reply:
Yea, this works for all the "presence" attributes. I am not sure why I added it. It works just the same without. I have remove the values now.